http/1.6/modules/ROOT/pages/http-authentication.adoc

```diff
@@ -989,30 +943,31 @@ This table includes examples of how to retrieve information from a Token Manager
 |Custom parameter extracted from the token URL response for resource owner `Peter` accessible through `vars.customParam` from DataWeave.
 |===
 
-=== Token Manager - Access Token Invalidation
-
-When using a Token Manager, you can block a particular resource owner.
+=== Configure Token Manager Access Token Invalidation
 
+When using a token manager, you can block a particular resource owner:
 
-=== Token Manager - Studio Visual Editor
+. In Studio, drag the OAuth module *Invalidate oauth context* operation to your flow.
+. In the properties editor, set the token manager to point to the same token manager that your HTTP *Request* operation references when managing OAuth authentication.
+. Set *Resource owner id* to an expression that points to the resource owner you want to clear, for example: `#[vars.resourceOwnerId]`
 
-. Drag an OAuth component's *Invalidate OAuth Context* operation to your canvas.
-. In its properties editor, set up the *Token Manager Configuration* so that it points to the same *Token Manager* that your HTTP Request Connector references when handling OAuth authentication.
-. Set the *Resource Owner Id* to an expression that points to the RO you want to clear. For example `#[vars.resourceOwnerId]`
+image::http-oauth-tokenmanager-2.png[Configure Token manager]
 
-=== Token Manager - XML Editor
+In the *Configuration XML* editor, the `invalidate-oauth-context` configuration look like this:
 
 [source,xml,linenums]
 ----
-<flow name="invalidateOauthContext">
-    <oauth:invalidate-oauth-context
-      tokenManager="tokenManagerConfig"/>
-</flow>
+  <flow name="invalidateOauthContext" >
+		<oauth:invalidate-oauth-context
+      tokenManager="tokenManagerConfig"
+       resourceOwnerId="#[vars.resourceOwnerId]">
+		</oauth:invalidate-oauth-context>
+	</flow>
 ----
 
-The *OAuth component's *Invalidate OAuth Context* operation cleans up all of the OAuth information stored in the token manager.
+The *Invalidate oauth context* operation cleans up all of the OAuth information stored in the token manager.
```

Comment:
"cleans up" = remove? reorganize? something else?
Try to avoid colocations: see style ref.